### ISSUE TITLE
Publish a public key for debian repo

### ISSUE THREAD

**Issue by ilya-epifanov** [sbt/sbt#2148]
There's no reference to a public key for .debs (and, probably .rpms, too) anywhere on the download page, as a result apt-get complains about sbt package.

**Comment by benmccann** [comment]
+1 i'm getting the same error

```
$ sudo apt-get update
W: GPG error: http://dl.bintray.com  Release: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 99E82A75642AC823
```

**Comment by eed3si9n** [comment]
Duplicate of sbt/sbt-launcher-package#72. Let me try auto signing.
